layouts/shortcodes/sign-language-section.html

```diff
@@ -0,0 +1,22 @@
+{{/*
+
+  Usage: {{% sign-language-section VIDEO_ID %}}
+
+  */ -}}
+
+{{ $youtube := "" -}}
+{{ $videoID := .Get 0 -}}
+{{ if $videoID -}}
+  {{ $youtube = printf "{{< youtube id=%q >}}" (printf "%s?mute=1" $videoID) }}
```

Comment:
@seokho-son - I've created #3416 to track the upgrade of Hugo in support of this PR, but I'm not sure we can upgrade Hugo without upgrading Docsy, and that will require time and effort. In the meantime I suggest a variant of what you proposed:

```suggestion
  {{ $youtube = printf "{{< youtube id=\"%s?mute=1\" >}}" $videoID -}}
```

This works for me.

Or we can just wait for the updated videos, though it might be nice to still have player muted (?).

Reply:
I've updated my sample PR with a solution pre- and post-Hugo 0.125.x upgrade, FYI: https://github.com/cncf/glossary/pull/3408/files#diff-146aa7a13ed8820b82b6ee313ade2cdc288094f71057f6a4145ebe478dfee6dc